.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,55 @@
+# How to contribute
```

Comment:
wouldn't it make more sense to provide a link to the Wiki page on contributing? https://github.com/adaptlearning/adapt_framework/wiki/Contributing-to-the-Adapt-Project